docs/TranslationWorkflow.md

```diff
@@ -0,0 +1,159 @@
+  element.parentNode.replaceChild(script, element)
+}
+
+function clean_document_download() {
+  const content = []
+  const lang = document.documentElement.getAttribute('lang');
+  const head = document.head.outerHTML
+    .replace(/<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi, '')
+    .replace(/<link rel="preload" as="(script|style)" href="[A-Za-z0-9\-\./]*\.(js|css)">/gi, '');
+  const button = document.getElementById(CSS_CLASS + '-DONE');
+  button.parentNode.removeChild(button);
+  const body = document.body.outerHTML
+    .replace(/<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi, '');
+  content.push('<!DOCTYPE html><html lang="' + lang + '">', head, body, '</html>');
+
+  download(content, FILENAME_PREFIX + '-index.html', 'text/html');
+}
+
+function add_section_highlighter() {
+  document.body.onclick = function(e) {
+    e.preventDefault();
+    if(e.target.classList.contains(CSS_CLASS)) {
+      var shouldDeselect = window.confirm('Unhighlight this section?');
+      if(shouldDeselect) e.target.classList.remove(CSS_CLASS);
+    } else {
+      e.target.classList.add(CSS_CLASS);
+      setTimeout(function(){
+        var shouldSelect = window.confirm('Highlight this section?');
+        if(!shouldSelect)  e.target.classList.remove(CSS_CLASS);
+      }, 200);
+    }
+  };
+}
+
+function add_file_builder() {
+  const a = document.createElement('a');
+  a.setAttribute('id', CSS_CLASS + '-DONE');
+  a.setAttribute('class', 'usa-button usa-button-big');
+  a.innerHTML = 'Translations Selected';
+  a.onclick = function(e) {
+    e.preventDefault();
+    e.stopPropagation();
+    let content = ["en,es,vi,ar\n"];
+    els = document.getElementsByClassName(CSS_CLASS);
+    Array.prototype.forEach.call(els, function(el, index) {
+      content = content.concat(['"'+el.outerHTML.replace(/"/g,'""')+'",,,\n']);
+      //js_downloads(el, index, FILENAME_PREFIX);
+    });
+    download(content, FILENAME_PREFIX + '.csv', 'text/csv');
+    clean_document_download();
+  }
+  document.body.appendChild(a);
+}
+
+document.addEventListener("DOMContentLoaded", function(event) { 
+  add_file_builder();
+  add_section_highlighter();
+});
+</script>
+```
+
+#### 4. Open your saved file in Chrome
+You should see a button at the top right that that reads *Translations Selected*
+
+#### 5. Highlight translations and download csv
+You can now click on items within the page to highlight them. This will indicate what sections need to be translated.
+As you click on items, you'll see a prompt which asks if you're sure about your selection. You may click an item to deselect it as well. 
+
+Once you've completed making selections, click the *Translations Selected* button. This will begin a download of two files. Your browser may ask for your permission to download multiple files. If so, click ok.
+
+Your files will be named something similar to **for-translations-1525922959252-index.html** and **for-translations-1525922959252.csv**
```

Comment:
Is there any possibility to make these names a little more descriptive? Perhaps the slug of the URL? Using the date and time?

Reply:
This is a great idea, but the export Chrome extension doesn't acknowledge the original filename, so I'd have to dig it up somewhere from the HTML. I'll investigate this and make updates (maybe use another chrome extension) and make a new PR if I find something simple to get the page slug.